bin/clover/src/spec/props.ts

```diff
@@ -118,10 +119,10 @@ function createPropFromCfInner(
     funcUniqueId: null,
     inputs: [],
     widgetKind: null,
-    widgetOptions: null,
+    widgetOptions: [],
     hidden: false,
     docLink: null,
-    documentation: null,
+    documentation: cfProp.description ?? null,
```

Comment:
@stack72 this adds prop docs

![image](https://github.com/user-attachments/assets/919d2fbe-6b4e-4212-ab1d-c25476b7a305)